### ISSUE TITLE
Support Unicode 11

### ISSUE THREAD

**Issue by mario** [mario/Emoji#7]

**Comment by vanniktech** [comment]
I think there was a limitation. We rely on emoji one for the ordering and they haven't pushed the latest changes yet. But there's more information one of the issues / PR on my repo.

**Comment by rubengees** [comment]
We are waiting for [this PR](https://github.com/iamcal/emoji-data/pull/146) to be merged.

**Comment by mario** [comment]
@vanniktech and @rubengees 

First of all, thanks for your answers and sorry it took me a bit to reply - super busy with a bunch  of things, so I really appreciate the time you've taken to write to me.

As for EmojiOne - from what I've seen, the datasources you use have dropped EmojiOne?

**Comment by vanniktech** [comment]
@rubengees can answer that. He did the amazing JS generation part.

**Comment by rubengees** [comment]
Yes, they have dropped EmojiOne by request of the EmojiOne team I think. The same would apply to us, if we were to update to the latest emojis - which is the correct way to go, since the EmojiOne seems to not want third parties to distribute their emoji set.

**Comment by vanniktech** [comment]
Yes. They also contacted me.

**Comment by vanniktech** [comment]
I replied but got no answer. I think it's best if we remove that module too.
